site/src/component/ChangelogModal/ChangelogModal.tsx

```diff
@@ -34,7 +35,10 @@ const ChangelogModal = () => {
       </Modal.Header>
 
       <p className="modal-body">{DESCRIPTION}</p>
-      <img className="modal-img" src={changelogImage} alt="Screenshot or gif of new changes" />
+      <div className="modal-side-by-side">
+        <img className="modal-img" src={changelogImage1} alt="Screenshot or gif of new changes" />
+        <img className="modal-img" src={changelogImage2} alt="Screenshot or gif of new changes" />
```

Comment:
On mobile this doesn't look good. We should hide the second image if the screen size dips below a certain amount.